src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java

```diff
@@ -14,6 +31,9 @@ public class DriveSubsystem extends SubsystemBase{
     private double rightTank = 0;
     private double arcadeForward = 0;
     private double arcadeRotation = 0;
+    
+    private boolean babyMode = false;
```

Comment:
Please open an Issue to allow babyMode to be toggled using a button on one of the joysticks